test_mq_service.py

```diff
@@ -125,7 +129,7 @@ def test_handle_message_happy_path(self, datetime_mock):
 
 
 @patch("app.queue.ingress.build_event")
-@patch("app.queue.ingress.add_host", return_value=(None, None))
+@patch("app.queue.ingress.add_host", return_value=({"id": "fAkE-iD"}, None))
```

Comment:
- We can even generate a random UUID here.
```suggestion
@patch("app.queue.ingress.add_host", return_value=({"id": str(uuid.uuid4())}, None))
```
- Or create _MagicMock_ instance.
```suggestion
@patch("app.queue.ingress.add_host", return_value=(MagicMock(), None))
```

Reply:
@Glutexo I went for the `MagicMock()` here!